"cobbler reposync" tests are broken on release33

Numerous issues have been seen in the CI with the first three `cobbler reposync` tests. This deserves some time before the 3.3.4 release.

Example: https://github.com/cobbler/cobbler/commit/7dfcece8ee40de7bc4bc237fc43acae397559748

<details>
  <summary>Click to see long test output</summary>

```
============================= test session starts ==============================
platform linux -- Python 3.6.15, pytest-3.10.1, py-1.10.0, pluggy-0.13.1
rootdir: /code, inifile: pytest.ini
plugins: mock-2.0.0, pythonpath-0.7.3, cov-2.8.1
collecting ... 
collecting 319 items                                                           
collecting 774 items                                                           
collecting 1012 items                                                          
collecting 1359 items                                                          
collected 1359 items                                                           

tests/autoinstallation_manager_test.py .                                 [  0%]
tests/configgen_test.py ......                                           [  0%]
tests/enums_test.py ....................                                 [  1%]
tests/grub_test.py ........                                              [  2%]
tests/module_loader_test.py ..............................               [  4%]
tests/templar_test.py ...s.                                              [  5%]
tests/template_api_test.py .......                                       [  5%]
tests/tftpgen_test.py ...                                                [  5%]
tests/utils_test.py ..........................................s.s....sss [  9%]
sss..........................................s........                   [ 13%]
tests/validate_test.py ................................................. [ 17%]
........                                                                 [ 17%]
tests/actions/buildiso_test.py ..........................                [ 19%]
tests/actions/mkloaders_test.py .......                                  [ 20%]
tests/actions/reposync_test.py FFF.......ssssss                          [ 21%]
tests/actions/sync_test.py sss                                           [ 21%]
tests/api/find_test.py ........................                          [ 23%]
tests/api/miscellaneous_test.py ...............................          [ 25%]
tests/api/sync_test.py ...................                               [ 27%]
tests/cli/cli_unit_test.py ...ssss...                                    [ 27%]
tests/cli/cobbler_cli_direct_test.py ..............s...s.                [ 29%]
tests/cli/cobbler_cli_object_test.py ................................... [ 31%]
......................................                                   [ 34%]
tests/collections/distro_collection_test.py ...........s                 [ 35%]
tests/items/distro_test.py ............................................. [ 38%]
...............                                                          [ 40%]
tests/items/file_test.py ...                                             [ 40%]
tests/items/image_test.py ......................                         [ 41%]
tests/items/item_test.py ...................................ss.......... [ 45%]
.....                                                                    [ 45%]
tests/items/menu_test.py ...                                             [ 45%]
tests/items/mgmtclass_test.py ........                                   [ 46%]
tests/items/network_interface_test.py .................................. [ 49%]
.....................................                                    [ 51%]
tests/items/package_test.py ....                                         [ 52%]
tests/items/profile_test.py ............................................ [ 55%]
.......................                                                  [ 56%]
tests/items/repo_test.py ..........................                      [ 58%]
tests/items/resource_test.py ........                                    [ 59%]
tests/items/system_test.py ............................................. [ 62%]
........................................................                 [ 66%]
tests/modules/nsupdate_add_system_post_test.py ..                        [ 67%]
tests/modules/nsupdate_delete_system_pre_test.py ..                      [ 67%]
tests/modules/scm_track_test.py ....                                     [ 67%]
tests/modules/sync_post_restart_services_test.py ..                      [ 67%]
tests/modules/sync_post_wingen_test.py ..                                [ 67%]
tests/modules/authentication/configfile_test.py .....                    [ 68%]
tests/modules/authentication/ldap_test.py ..........                     [ 68%]
tests/modules/authentication/pam_test.py .                               [ 68%]
tests/modules/authentication/passthrough_test.py ..                      [ 69%]
tests/modules/authorization/allowall_test.py ..                          [ 69%]
tests/modules/authorization/configfile_test.py .                         [ 69%]
tests/modules/authorization/ownership_test.py .                          [ 69%]
tests/modules/installation/post_log_test.py ..                           [ 69%]
tests/modules/installation/post_power_test.py ..                         [ 69%]
tests/modules/installation/post_puppet_test.py ..                        [ 69%]
tests/modules/installation/post_report_test.py .s                        [ 69%]
tests/modules/installation/pre_clear_anamon_logs_test.py ..              [ 70%]
tests/modules/installation/pre_log_test.py ..                            [ 70%]
tests/modules/installation/pre_puppet_test.py ..                         [ 70%]
tests/modules/managers/bind_test.py ...                                  [ 70%]
tests/modules/managers/dnsmasq_test.py .......                           [ 71%]
tests/modules/managers/genders_test.py ...                               [ 71%]
tests/modules/managers/import_signatures_test.py .s..                    [ 71%]
tests/modules/managers/in_tftpd_test.py ...s.....                        [ 72%]
tests/modules/managers/isc_test.py ........                              [ 72%]
tests/modules/managers/ndjbdns_test.py ....                              [ 73%]
tests/modules/serializer/file_test.py .................                  [ 74%]
tests/settings/settings_test.py ........                                 [ 75%]
tests/settings/migrations/helper_test.py ........                        [ 75%]
tests/settings/migrations/migrations_test.py ...............             [ 76%]
tests/settings/migrations/normalize_test.py ...........                  [ 77%]
tests/special_cases/security_test.py ....                                [ 77%]
tests/xmlrpcapi/background_test.py .......                               [ 78%]
tests/xmlrpcapi/distro_test.py ...............................           [ 80%]
tests/xmlrpcapi/file_test.py .......                                     [ 81%]
tests/xmlrpcapi/image_test.py .......                                    [ 81%]
tests/xmlrpcapi/item_test.py .                                           [ 81%]
tests/xmlrpcapi/menu_test.py ........                                    [ 82%]
tests/xmlrpcapi/mgmt_class_test.py .......                               [ 82%]
tests/xmlrpcapi/miscellaneous_test.py ....s........s.................... [ 85%]
s                                                                        [ 85%]
tests/xmlrpcapi/non_object_calls_test.py ........................        [ 87%]
tests/xmlrpcapi/package_test.py .......                                  [ 87%]
tests/xmlrpcapi/profile_test.py ........................................ [ 90%]
............................................                             [ 93%]
tests/xmlrpcapi/repo_test.py .......                                     [ 94%]
tests/xmlrpcapi/system_test.py ......................................... [ 97%]
...................................                                      [100%]

=================================== FAILURES ===================================
_ TestRepoSync.test_reposync_yum[MirrorType.BASEURL-http://download.fedoraproject.org/pub/fedora/linux/releases/35/Everything/x86_64/os-expected_exception0] _

self = <tests.actions.reposync_test.TestRepoSync object at 0x7f8c9ed16cc0>
input_mirror_type = <MirrorType.BASEURL: 'baseurl'>
input_mirror = 'http://download.fedoraproject.org/pub/fedora/linux/releases/35/Everything/x86_64/os'
expected_exception = <contextlib._GeneratorContextManager object at 0x7f8ca1db3080>
api = <cobbler.api.CobblerAPI object at 0x7f8c9ed16f98>
repo = <cobbler.items.repo.Repo object at 0x7f8c9ecfdb70>
reposync = <cobbler.actions.reposync.RepoSync object at 0x7f8c9ed16f60>

    @pytest.mark.usefixtures("remove_repo")
    @pytest.mark.parametrize(
        "input_mirror_type,input_mirror,expected_exception",
        [
            (
                enums.MirrorType.BASEURL,
                "http://download.fedoraproject.org/pub/fedora/linux/releases/35/Everything/x86_64/os",
                does_not_raise()
            ),
            (
                enums.MirrorType.MIRRORLIST,
                "https://mirrors.fedoraproject.org/mirrorlist?repo=fedora-35&arch=x86_64",
                does_not_raise()
            ),
            (
                enums.MirrorType.METALINK,
                "https://mirrors.fedoraproject.org/metalink?repo=fedora-35&arch=x86_64",
                does_not_raise()
            ),
            (
                enums.MirrorType.BASEURL,
                "http://www.example.com/path/to/some/repo",
                pytest.raises(cexceptions.CX)
            ),
        ],
    )
    def test_reposync_yum(
        self,
        input_mirror_type,
        input_mirror,
        expected_exception,
        api,
        repo,
        reposync
    ):
        # Arrange
        test_repo = repo
        test_repo.breed = enums.RepoBreeds.YUM
        test_repo.mirror = input_mirror
        test_repo.mirror_type = input_mirror_type
        test_repo.rpm_list = "fedora-gpg-keys"
        test_settings = api.settings()
        repo_path = os.path.join(test_settings.webdir, "repo_mirror", test_repo.name)
    
        # Act & Assert
        with expected_exception:
>           reposync.run(test_repo.name)

tests/actions/reposync_test.py:96: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <cobbler.actions.reposync.RepoSync object at 0x7f8c9ed16f60>
name = 'testrepo0', verbose = True

    def run(self, name: Optional[str] = None, verbose: bool = True):
        """
        Syncs the current repo configuration file with the filesystem.
    
        :param name: The name of the repository to synchronize.
        :param verbose: If the action should be logged verbose or not.
        """
    
        self.logger.info("run, reposync, run!")
    
        self.verbose = verbose
        report_failure = False
        for repo in self.repos:
            if name is not None and repo.name != name:
                # Invoked to sync only a specific repo, this is not the one
                continue
            elif name is None and not repo.keep_updated:
                # Invoked to run against all repos, but this one is off
                self.logger.info("%s is set to not be updated", repo.name)
                continue
    
            repo_mirror = os.path.join(self.settings.webdir, "repo_mirror")
            repo_path = os.path.join(repo_mirror, repo.name)
    
            if not os.path.isdir(repo_path) and not repo.mirror.lower().startswith("rhn://"):
                os.makedirs(repo_path)
    
            # Set the environment keys specified for this repo and save the old one if they modify an existing variable.
    
            env = repo.environment
            old_env = {}
    
            for k in list(env.keys()):
                self.logger.debug("setting repo environment: %s=%s", k, env[k])
                if env[k] is not None:
                    if os.getenv(k):
                        old_env[k] = os.getenv(k)
                    else:
                        os.environ[k] = env[k]
    
            # Which may actually NOT reposync if the repo is set to not mirror locally but that's a technicality.
    
            for x in range(self.tries + 1, 1, -1):
                success = False
                try:
                    self.sync(repo)
                    success = True
                    break
                except:
                    utils.log_exc()
                    self.logger.warning("reposync failed, tries left: %s", (x - 2))
    
            # Cleanup/restore any environment variables that were added or changed above.
    
            for k in list(env.keys()):
                if env[k] is not None:
                    if k in old_env:
                        self.logger.debug(
                            "resetting repo environment: %s=%s", k, old_env[k]
                        )
                        os.environ[k] = old_env[k]
                    else:
                        self.logger.debug("removing repo environment: %s=%s", k, env[k])
                        del os.environ[k]
    
            if not success:
                report_failure = True
                if not self.nofail:
>                   raise CX("reposync failed, retry limit reached, aborting")
E                   cobbler.cexceptions.CX: 'reposync failed, retry limit reached, aborting'

cobbler/actions/reposync.py:173: CX
------------------------------ Captured log call -------------------------------
reposync.py                155 WARNING  reposync failed, tries left: 1
reposync.py                155 WARNING  reposync failed, tries left: 0
_ TestRepoSync.test_reposync_yum[MirrorType.MIRRORLIST-https://mirrors.fedoraproject.org/mirrorlist?repo=fedora-35&arch=x86_64-expected_exception1] _

self = <tests.actions.reposync_test.TestRepoSync object at 0x7f8c9ed16860>
input_mirror_type = <MirrorType.MIRRORLIST: 'mirrorlist'>
input_mirror = 'https://mirrors.fedoraproject.org/mirrorlist?repo=fedora-35&arch=x86_64'
expected_exception = <contextlib._GeneratorContextManager object at 0x7f8ca1db3240>
api = <cobbler.api.CobblerAPI object at 0x7f8c9ed16f98>
repo = <cobbler.items.repo.Repo object at 0x7f8c9ed16dd8>
reposync = <cobbler.actions.reposync.RepoSync object at 0x7f8c9ed16f60>

    @pytest.mark.usefixtures("remove_repo")
    @pytest.mark.parametrize(
        "input_mirror_type,input_mirror,expected_exception",
        [
            (
                enums.MirrorType.BASEURL,
                "http://download.fedoraproject.org/pub/fedora/linux/releases/35/Everything/x86_64/os",
                does_not_raise()
            ),
            (
                enums.MirrorType.MIRRORLIST,
                "https://mirrors.fedoraproject.org/mirrorlist?repo=fedora-35&arch=x86_64",
                does_not_raise()
            ),
            (
                enums.MirrorType.METALINK,
                "https://mirrors.fedoraproject.org/metalink?repo=fedora-35&arch=x86_64",
                does_not_raise()
            ),
            (
                enums.MirrorType.BASEURL,
                "http://www.example.com/path/to/some/repo",
                pytest.raises(cexceptions.CX)
            ),
        ],
    )
    def test_reposync_yum(
        self,
        input_mirror_type,
        input_mirror,
        expected_exception,
        api,
        repo,
        reposync
    ):
        # Arrange
        test_repo = repo
        test_repo.breed = enums.RepoBreeds.YUM
        test_repo.mirror = input_mirror
        test_repo.mirror_type = input_mirror_type
        test_repo.rpm_list = "fedora-gpg-keys"
        test_settings = api.settings()
        repo_path = os.path.join(test_settings.webdir, "repo_mirror", test_repo.name)
    
        # Act & Assert
        with expected_exception:
>           reposync.run(test_repo.name)

tests/actions/reposync_test.py:96: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <cobbler.actions.reposync.RepoSync object at 0x7f8c9ed16f60>
name = 'testrepo0', verbose = True

    def run(self, name: Optional[str] = None, verbose: bool = True):
        """
        Syncs the current repo configuration file with the filesystem.
    
        :param name: The name of the repository to synchronize.
        :param verbose: If the action should be logged verbose or not.
        """
    
        self.logger.info("run, reposync, run!")
    
        self.verbose = verbose
        report_failure = False
        for repo in self.repos:
            if name is not None and repo.name != name:
                # Invoked to sync only a specific repo, this is not the one
                continue
            elif name is None and not repo.keep_updated:
                # Invoked to run against all repos, but this one is off
                self.logger.info("%s is set to not be updated", repo.name)
                continue
    
            repo_mirror = os.path.join(self.settings.webdir, "repo_mirror")
            repo_path = os.path.join(repo_mirror, repo.name)
    
            if not os.path.isdir(repo_path) and not repo.mirror.lower().startswith("rhn://"):
                os.makedirs(repo_path)
    
            # Set the environment keys specified for this repo and save the old one if they modify an existing variable.
    
            env = repo.environment
            old_env = {}
    
            for k in list(env.keys()):
                self.logger.debug("setting repo environment: %s=%s", k, env[k])
                if env[k] is not None:
                    if os.getenv(k):
                        old_env[k] = os.getenv(k)
                    else:
                        os.environ[k] = env[k]
    
            # Which may actually NOT reposync if the repo is set to not mirror locally but that's a technicality.
    
            for x in range(self.tries + 1, 1, -1):
                success = False
                try:
                    self.sync(repo)
                    success = True
                    break
                except:
                    utils.log_exc()
                    self.logger.warning("reposync failed, tries left: %s", (x - 2))
    
            # Cleanup/restore any environment variables that were added or changed above.
    
            for k in list(env.keys()):
                if env[k] is not None:
                    if k in old_env:
                        self.logger.debug(
                            "resetting repo environment: %s=%s", k, old_env[k]
                        )
                        os.environ[k] = old_env[k]
                    else:
                        self.logger.debug("removing repo environment: %s=%s", k, env[k])
                        del os.environ[k]
    
            if not success:
                report_failure = True
                if not self.nofail:
>                   raise CX("reposync failed, retry limit reached, aborting")
E                   cobbler.cexceptions.CX: 'reposync failed, retry limit reached, aborting'

cobbler/actions/reposync.py:173: CX
------------------------------ Captured log call -------------------------------
reposync.py                155 WARNING  reposync failed, tries left: 1
reposync.py                155 WARNING  reposync failed, tries left: 0
_ TestRepoSync.test_reposync_yum[MirrorType.METALINK-https://mirrors.fedoraproject.org/metalink?repo=fedora-35&arch=x86_64-expected_exception2] _

self = <tests.actions.reposync_test.TestRepoSync object at 0x7f8c9ed95860>
input_mirror_type = <MirrorType.METALINK: 'metalink'>
input_mirror = 'https://mirrors.fedoraproject.org/metalink?repo=fedora-35&arch=x86_64'
expected_exception = <contextlib._GeneratorContextManager object at 0x7f8ca1db32b0>
api = <cobbler.api.CobblerAPI object at 0x7f8c9ed16f98>
repo = <cobbler.items.repo.Repo object at 0x7f8c9ed79dd8>
reposync = <cobbler.actions.reposync.RepoSync object at 0x7f8c9ed16f60>

    @pytest.mark.usefixtures("remove_repo")
    @pytest.mark.parametrize(
        "input_mirror_type,input_mirror,expected_exception",
        [
            (
                enums.MirrorType.BASEURL,
                "http://download.fedoraproject.org/pub/fedora/linux/releases/35/Everything/x86_64/os",
                does_not_raise()
            ),
            (
                enums.MirrorType.MIRRORLIST,
                "https://mirrors.fedoraproject.org/mirrorlist?repo=fedora-35&arch=x86_64",
                does_not_raise()
            ),
            (
                enums.MirrorType.METALINK,
                "https://mirrors.fedoraproject.org/metalink?repo=fedora-35&arch=x86_64",
                does_not_raise()
            ),
            (
                enums.MirrorType.BASEURL,
                "http://www.example.com/path/to/some/repo",
                pytest.raises(cexceptions.CX)
            ),
        ],
    )
    def test_reposync_yum(
        self,
        input_mirror_type,
        input_mirror,
        expected_exception,
        api,
        repo,
        reposync
    ):
        # Arrange
        test_repo = repo
        test_repo.breed = enums.RepoBreeds.YUM
        test_repo.mirror = input_mirror
        test_repo.mirror_type = input_mirror_type
        test_repo.rpm_list = "fedora-gpg-keys"
        test_settings = api.settings()
        repo_path = os.path.join(test_settings.webdir, "repo_mirror", test_repo.name)
    
        # Act & Assert
        with expected_exception:
>           reposync.run(test_repo.name)

tests/actions/reposync_test.py:96: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <cobbler.actions.reposync.RepoSync object at 0x7f8c9ed16f60>
name = 'testrepo0', verbose = True

    def run(self, name: Optional[str] = None, verbose: bool = True):
        """
        Syncs the current repo configuration file with the filesystem.
    
        :param name: The name of the repository to synchronize.
        :param verbose: If the action should be logged verbose or not.
        """
    
        self.logger.info("run, reposync, run!")
    
        self.verbose = verbose
        report_failure = False
        for repo in self.repos:
            if name is not None and repo.name != name:
                # Invoked to sync only a specific repo, this is not the one
                continue
            elif name is None and not repo.keep_updated:
                # Invoked to run against all repos, but this one is off
                self.logger.info("%s is set to not be updated", repo.name)
                continue
    
            repo_mirror = os.path.join(self.settings.webdir, "repo_mirror")
            repo_path = os.path.join(repo_mirror, repo.name)
    
            if not os.path.isdir(repo_path) and not repo.mirror.lower().startswith("rhn://"):
                os.makedirs(repo_path)
    
            # Set the environment keys specified for this repo and save the old one if they modify an existing variable.
    
            env = repo.environment
            old_env = {}
    
            for k in list(env.keys()):
                self.logger.debug("setting repo environment: %s=%s", k, env[k])
                if env[k] is not None:
                    if os.getenv(k):
                        old_env[k] = os.getenv(k)
                    else:
                        os.environ[k] = env[k]
    
            # Which may actually NOT reposync if the repo is set to not mirror locally but that's a technicality.
    
            for x in range(self.tries + 1, 1, -1):
                success = False
                try:
                    self.sync(repo)
                    success = True
                    break
                except:
                    utils.log_exc()
                    self.logger.warning("reposync failed, tries left: %s", (x - 2))
    
            # Cleanup/restore any environment variables that were added or changed above.
    
            for k in list(env.keys()):
                if env[k] is not None:
                    if k in old_env:
                        self.logger.debug(
                            "resetting repo environment: %s=%s", k, old_env[k]
                        )
                        os.environ[k] = old_env[k]
                    else:
                        self.logger.debug("removing repo environment: %s=%s", k, env[k])
                        del os.environ[k]
    
            if not success:
                report_failure = True
                if not self.nofail:
>                   raise CX("reposync failed, retry limit reached, aborting")
E                   cobbler.cexceptions.CX: 'reposync failed, retry limit reached, aborting'

cobbler/actions/reposync.py:173: CX
------------------------------ Captured log call -------------------------------
reposync.py                155 WARNING  reposync failed, tries left: 1
reposync.py                155 WARNING  reposync failed, tries left: 0

---------- coverage: platform linux, python 3.6.15-final-0 -----------
Name                                                    Stmts   Miss  Cover
---------------------------------------------------------------------------
cobbler/__init__.py                                         0      0   100%
cobbler/actions/__init__.py                                 0      0   100%
cobbler/actions/acl.py                                     48     32    33%
cobbler/actions/buildiso/__init__.py                      151     20    87%
cobbler/actions/buildiso/netboot.py                       306    171    44%
cobbler/actions/buildiso/standalone.py                    127     82    35%
cobbler/actions/check.py                                  229    101    56%
cobbler/actions/hardlink.py                                28      4    86%
cobbler/actions/log.py                                     25      8    68%
cobbler/actions/mkloaders.py                              109     18    83%
cobbler/actions/replicate.py                              234    194    17%
cobbler/actions/report.py                                 200    184     8%
cobbler/actions/reposync.py                               427    276    35%
cobbler/actions/status.py                                  99     63    36%
cobbler/actions/sync.py                                   246     65    74%
cobbler/api.py                                            627    226    64%
cobbler/autoinstall_manager.py                            162     53    67%
cobbler/autoinstallgen.py                                 177    157    11%
cobbler/cexceptions.py                                      8      0   100%
cobbler/cli.py                                            589    205    65%
cobbler/cobbler_collections/__init__.py                     0      0   100%
cobbler/cobbler_collections/collection.py                 225     60    73%
cobbler/cobbler_collections/distros.py                     61     13    79%
cobbler/cobbler_collections/files.py                       33      4    88%
cobbler/cobbler_collections/images.py                      43      7    84%
cobbler/cobbler_collections/manager.py                     88      7    92%
cobbler/cobbler_collections/menus.py                       45      9    80%
cobbler/cobbler_collections/mgmtclasses.py                 32      4    88%
cobbler/cobbler_collections/packages.py                    32      4    88%
cobbler/cobbler_collections/profiles.py                    48      6    88%
cobbler/cobbler_collections/repos.py                       36      4    89%
cobbler/cobbler_collections/systems.py                     38      4    89%
cobbler/cobblerd.py                                        49     49     0%
cobbler/configgen.py                                       71     33    54%
cobbler/decorator.py                                        3      0   100%
cobbler/download_manager.py                                26     14    46%
cobbler/enums.py                                          113      0   100%
cobbler/grub.py                                            22      0   100%
cobbler/items/__init__.py                                   0      0   100%
cobbler/items/distro.py                                   195     14    93%
cobbler/items/file.py                                      39      6    85%
cobbler/items/image.py                                    218     34    84%
cobbler/items/item.py                                     404     52    87%
cobbler/items/menu.py                                      65     11    83%
cobbler/items/mgmtclass.py                                 64      6    91%
cobbler/items/package.py                                   34      2    94%
cobbler/items/profile.py                                  335     16    95%
cobbler/items/repo.py                                     196     28    86%
cobbler/items/resource.py                                  68      5    93%
cobbler/items/system.py                                   900    112    88%
cobbler/manager.py                                         20      1    95%
cobbler/module_loader.py                                   78     11    86%
cobbler/modules/__init__.py                                 0      0   100%
cobbler/modules/authentication/__init__.py                  0      0   100%
cobbler/modules/authentication/configfile.py               50      9    82%
cobbler/modules/authentication/denyall.py                   4      1    75%
cobbler/modules/authentication/ldap.py                     85      8    91%
cobbler/modules/authentication/pam.py                      67      6    91%
cobbler/modules/authentication/passthru.py                  5      0   100%
cobbler/modules/authentication/spacewalk.py                38     29    24%
cobbler/modules/authorization/__init__.py                   0      0   100%
cobbler/modules/authorization/allowall.py                   4      0   100%
cobbler/modules/authorization/configfile.py                25     17    32%
cobbler/modules/authorization/ownership.py                 98     88    10%
cobbler/modules/installation/__init__.py                    0      0   100%
cobbler/modules/installation/post_log.py                   17      3    82%
cobbler/modules/installation/post_power.py                 22      7    68%
cobbler/modules/installation/post_puppet.py                33     19    42%
cobbler/modules/installation/post_report.py                49     38    22%
cobbler/modules/installation/pre_clear_anamon_logs.py      24      5    79%
cobbler/modules/installation/pre_log.py                    17      3    82%
cobbler/modules/installation/pre_puppet.py                 37     23    38%
cobbler/modules/managers/__init__.py                        0      0   100%
cobbler/modules/managers/bind.py                          338    305    10%
cobbler/modules/managers/dnsmasq.py                        93     12    87%
cobbler/modules/managers/genders.py                        58      8    86%
cobbler/modules/managers/import_signatures.py             480    422    12%
cobbler/modules/managers/in_tftpd.py                       99     22    78%
cobbler/modules/managers/isc.py                           220    104    53%
cobbler/modules/managers/ndjbdns.py                        40      2    95%
cobbler/modules/nsupdate_add_system_post.py                82     50    39%
cobbler/modules/nsupdate_delete_system_pre.py              82     48    41%
cobbler/modules/scm_track.py                               43      2    95%
cobbler/modules/serializers/__init__.py                     0      0   100%
cobbler/modules/serializers/file.py                        62      6    90%
cobbler/modules/serializers/mongodb.py                     65     45    31%
cobbler/modules/sync_post_restart_services.py              40     17    58%
cobbler/modules/sync_post_wingen.py                       251    206    18%
cobbler/power_manager.py                                  120     81    32%
cobbler/remote.py                                        1368    534    61%
cobbler/serializer.py                                      53      7    87%
cobbler/services.py                                       204    204     0%
cobbler/settings/__init__.py                              243     25    90%
cobbler/settings/migrations/V2_8_5.py                      14      4    71%
cobbler/settings/migrations/V3_0_0.py                      31      3    90%
cobbler/settings/migrations/V3_0_1.py                      15      3    80%
cobbler/settings/migrations/V3_1_0.py                      15      3    80%
cobbler/settings/migrations/V3_1_1.py                      15      3    80%
cobbler/settings/migrations/V3_1_2.py                      17      3    82%
cobbler/settings/migrations/V3_2_0.py                      20      1    95%
cobbler/settings/migrations/V3_2_1.py                      28      5    82%
cobbler/settings/migrations/V3_3_0.py                      32      3    91%
cobbler/settings/migrations/V3_3_1.py                      20      3    85%
cobbler/settings/migrations/V3_3_2.py                      15      3    80%
cobbler/settings/migrations/V3_3_3.py                      16      3    81%
cobbler/settings/migrations/__init__.py                   137     42    69%
cobbler/settings/migrations/helper.py                      60      3    95%
cobbler/templar.py                                        113     40    65%
cobbler/template_api.py                                    76     19    75%
cobbler/tftpgen.py                                        755    332    56%
cobbler/utils.py                                          973    283    71%
cobbler/validate.py                                       280     45    84%
cobbler/yumgen.py                                          35     18    49%
---------------------------------------------------------------------------
TOTAL                                                   14556   5550    62%

============= 3 failed, 1322 passed, 34 skipped in 296.83 seconds ==============
Error: Process completed with exit code 1.
```

</details>

--- a/.gitignore
+++ b/.gitignore
@@ -41,6 +41,8 @@ test.log
 venv/
 # Venv for developnig with Python3 for the futurize branch.
 venv3/
+# Vscode
+.vscode
 
 # Coverage
 covhtml/

--- a/.obs/workflows.yml
+++ b/.obs/workflows.yml
@@ -1,6 +1,10 @@
 workflow:
   steps:
-    - branch_package:
-        source_project: systemsmanagement:orthos2:github_master_ci
-        source_package: cobbler-master
-        target_project: home:dgedon
+    - trigger_services:
+        project: home:cobbler-project:github-ci:release33
+        package: cobbler-docker-testing
+  filters:
+    event: push
+    branches:
+      only:
+        - release33

--- a/docker/develop/develop.dockerfile
+++ b/docker/develop/develop.dockerfile
@@ -83,7 +83,7 @@ RUN zypper install --no-recommends -y \
     shim \
     ipxe-bootimgs \
     grub2 \
-    grub2-i386-efi \
+#    grub2-i386-efi \
     grub2-x86_64-efi \
     grub2-arm64-efi \
     && zypper clean
@@ -130,7 +130,7 @@ RUN zypper install --no-recommends -y \
 RUN sed -i 's/DHCPD_INTERFACE=""/DHCPD_INTERFACE="ANY"/' /etc/sysconfig/dhcpd
 
 # Add Testuser for the PAM tests
-RUN useradd -p "$(perl -e 'print crypt(\"test\", \"password\")')" test
+RUN useradd -p "$(perl -e 'print crypt("test", "password")')" test
 
 # Add Developer scripts to PATH
 ENV PATH="/code/docker/develop/scripts:${PATH}"

